pages/leaderboard.tsx

```diff
@@ -149,44 +157,145 @@ export default function Leaderboard({ loginContext }: Props) {
+                'mb-4',
+                'md:mb-8',
+                'md:w-1/2'
+              )}
+            >
+              <div
+                className={clsx(
+                  'border-r',
+                  'border-black',
+                  'flex',
+                  'h-full',
+                  'items-center',
+                  'justify-between',
+                  'w-1/2'
                 )}
-              </label>
-            </div>
-            <div className="h-full flex items-center justify-between w-1/4">
-              <label className="flex flex-col font-favorit text-xs px-2.5 w-full">
-                View:
-                {$eventType && $eventType.value && (
-                  <Select {...$eventType} className="text-lg" />
+              >
+                <label
+                  className={clsx(
+                    'flex',
+                    'flex-col',
+                    'font-favorit',
+                    'text-xs',
+                    'px-2.5',
+                    'w-full'
+                  )}
+                >
+                  Region:
+                  {$country && $country.value && (
+                    <Select {...$country} className="text-lg" />
+                  )}
+                </label>
+              </div>
+              <div
+                className={clsx(
+                  'h-full',
+                  'flex',
+                  'items-center',
+                  'justify-between',
+                  'w-1/2'
                 )}
-              </label>
+              >
+                <label
+                  className={clsx(
+                    'flex',
+                    'flex-col',
+                    'font-favorit',
+                    'text-xs',
+                    'px-2.5',
+                    'w-full'
+                  )}
+                >
+                  View:
+                  {$eventType && $eventType.value && (
+                    <Select {...$eventType} className="text-lg" />
+                  )}
+                </label>
+              </div>
             </div>
           </div>
-          <div className="font-extended flex px-4 sm:px-10 mb-4">
+          <div
+            className={clsx(
+              'font-extended',
+              'flex',
+              'px-4',
+              'sm:px-10',
+              'mb-4'
+            )}
+          >
             {$users.length > 0 && (
               <>
-                <div className="w-16 sm:w-24 hidden md:inline">RANK</div>
-                <div className="flex-1 hidden md:inline">USERNAME</div>
-                <div className="ml-2  hidden md:inline">TOTAL POINTS</div>
+                <div className={clsx('w-16', 'sm:w-24', 'hidden', 'md:inline')}>
+                  RANK
+                </div>
+                <div className={clsx('flex-1', 'hidden', 'md:inline')}>
+                  USERNAME
+                </div>
+                <div className={clsx('ml-2', '', 'hidden', 'md:inline')}>
```

Comment:
The extra space can probably be removed here, unless it's doing something I don't realize.

Reply:
Good catch, I wrote a vim script to convert raw `className` strings to `clsx` entries, but it doesn't currently catch double-spaces. Will amend.